dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -343,7 +343,13 @@ It should only be enabled for the `ws_3D` and `ws_3D_QLab` workspaces (see comme
 
     - Check that the table values are updated correctly (use the HKL in the cursor info table to help determine this)
 
-27. Produce a non-axis aligned cut where the peak is not in the center of the x-range. Take the x-value at the peak maximum and plug it into the axis label - it should produce the HKL of the peak (0,0,1).
+27. Produce a non-axis aligned cut as shown below where the peak is not in the center of the x-range. Take the x-value at the peak maximum and plug it into the axis label - it should produce the HKL of the peak (0,0,1).
+
+    - In the example below, the x axis label is ``(0.0, -0.24 + 0.97x, 0.93 + 0.25x) in 7.07 Ang^-1``. So if we plug in the value of x at the peak centre we get ``(0.0, -0.24 + 0.97*0.25, 0.93 + 0.25*0.25)`` = ``(0.0, 0.0025, 0.9925)`` which is very close actual peak HKL = ``(0,0,1)``
```

Comment:
This is a nitpick, so I'm happy if you'd prefer to disregard this, but using `:math:` in the `rst` will make the documentation more closely match the image described: 
 
```suggestion
    - In the example below, the x axis label is :math:`(0.0, -0.24 + 0.97x, 0.93 + 0.25x) in 7.07 \AA^{-1}`. So if we plug in the value of x at the peak centre we get ``(0.0, -0.24 + 0.97*0.25, 0.93 + 0.25*0.25)`` = ``(0.0, 0.0025, 0.9925)``, which is very close actual peak HKL = ``(0,0,1)``
```